source/tasks/Utils/command-line-args.test.ts

```diff
@@ -0,0 +1,52 @@
+import commandLineArgs from "command-line-args";
+import shlex from "shlex";
+
+describe("cli args", () => {
```

Comment:
These tests were me working out how to parse the arguments from the `AdditionalArguments` field, thought I would leave them here for future reference